resources.html

```diff
@@ -99,7 +99,7 @@ <h6>Project Resources</h6>
                     <a href="https://infrafinder.investinopen.org/solutions/public-access-submission-system" target="_blank" rel="noopener noreferrer">Invest in Open Infrastructure Infra Finder entry</a>
                   </li>
                   <li>
-                    <a href="https://eclipse-pass.org/main/docs/roadmap" target="_blank" rel="noopener noreferrer">Project Roadmap</a>
+                    <a href="https://docs.eclipse-pass.org/pass-documentation-dev/community/pass-roadmap" target="_blank" rel="noopener noreferrer">Project Roadmap</a>
```

Comment:
This link points to the dev docs. Before this is merged, it should be updated to point to the production URL.